uvicorn/server_state.py

```diff
@@ -0,0 +1,16 @@
+import asyncio
```

Comment:
I had to move this due to circular import on the `WebProtocol`. :cry:

Reply:
11 hours after doing this, I wonder if this is a good idea. It's a breaking change as we were exposing `ServerState` on `server.py`. I can't find a better solution, so suggestions are welcome.